src/viperleed/calc/files/parameters/file_reader.py

```diff
@@ -0,0 +1,61 @@
+"""Module file_reader of viperleed.calc.classes.
+
+This module is a more abstract version of a settings file reader. It is based on
```

Comment:
```suggestion
This module is a more abstract version of an input-file reader. It is based on
```

(See my suggestion below concerning naming of the class.)